release-notes/5.0/5.0.0-preview3.md

```diff
@@ -0,0 +1,91 @@
+# Release Notes
+
+## Microsoft.Data.SqlClient 5.0.0-preview3 released 15 June 2022
+
+This update brings the below changes over the previous release:
+
+### Added
+- Added support for `TDS 8`. To use TDS 8, users should specify Encrypt=Strict in the connection string. Strict mode disables TrustServerCertificate (always treated as False in Strict mode). HostNameInCertificate has been added to help some Strict mode scenarios. [#1608](https://github.com/dotnet/SqlClient/pull/1608)
```

Comment:
Please add a section with sample connection strings with additional descriptions and link it here.
```suggestion
- Increased the security by adding support of `TDS 8`. [#1608](https://github.com/dotnet/SqlClient/pull/1608)
```